public/emails/invited_to_org.txt

```diff
@@ -0,0 +1,10 @@
+{{Subject .Subject "{{.InvitedBy}} has added you to the {{.OrgName}} organization"}}
+
+You have been added to {{.OrgName}}
+
+{{.InvitedBy}} has added you to the {{.OrgName}} organization in Grafana.
+Once logged in, {{.OrgName}} will be available in the left side menu, in the dropdown
```

Comment:
```suggestion
Once logged in, {{.OrgName}} will be available in the left side menu, in the drop-down list
```

Reply:
@oddlittlebird Note that I have just copied the contents from the HTML templates so if you want this to be changed here, it should be changed in the HTML as well.

Reply:
Fixed typos but did not use Markdown syntax